java/lib/src/main/java/com/svix/Endpoint.java

```diff
@@ -94,7 +98,8 @@ public void recover(final String appId, final String endpointId, final RecoverIn
 		this.recover(appId, endpointId, recoverIn, new PostOptions());
 	}
 
-	public void recover(final String appId, final String endpointId, final RecoverIn recoverIn, final PostOptions options) throws ApiException {
+	public void recover(final String appId, final String endpointId, final RecoverIn recoverIn,
```

Comment:
Why did you add options to all of these? Also, isn't this a breaking change? Now these options are required?

Reply:
With the exception of the new overloaded `getStats` method, I think these other changes are just reformats my code editor did automatically.

I can undo them though, I didn't think they were a big deal.